<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updater</class>
 <widget class="QDialog" name="Updater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>363</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Certificate update</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
border-radius: 2px;
background-color: #FFFFFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="UpdaterLayout" stretch="1,0,0">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="processPage">
      <layout class="QVBoxLayout" name="processPageLayout" stretch="0,0,0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="log">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit {
	border: 1px solid #DEE4E9;
	border-radius: 2px;
	background-color: #FFFFFF;
}
QScrollBar {
	width: 10px;
	padding: 5px 5px 5px 0;
	margin: 2px 0px 2px 0;
	border: none;
	background: #FFFFFF;
}
QScrollBar::handle:vertical{
	background-color: #DEE4E9;
	margin: 0 0 0 0;
	border-radius: 2px;
	height: 145px;
}
QScrollBar::add-line:vertical {
      border: none;
      background: none;
}
QScrollBar::sub-line:vertical {
      border: none;
      background: none;
}</string>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="progressRunning">
         <property name="text">
          <string>Validity extension in progress. The process may take up to 10 minutes. Do not remove the card from reader!</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMessage">
      <layout class="QVBoxLayout" name="pageMessageLayout" stretch="0,1,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblHeader">
         <property name="styleSheet">
          <string notr="true">font-family: Open Sans;
font-size: 16px;
color: #000000;
font-weight: 700;
</string>
         </property>
         <property name="text">
          <string>Validity Extension</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="message">
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="messageAgree">
         <property name="text">
          <string>I agree to the terms of use of certificates.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="envelopePage">
      <layout class="QVBoxLayout" name="envelopePageLayout" stretch="0,1,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="envelopeLabel"/>
       </item>
       <item>
        <widget class="QLabel" name="envelope"/>
       </item>
       <item>
        <widget class="QCheckBox" name="envelopeAgree">
         <property name="text">
          <string>I confirm I have saved new PIN1, PIN2 and PUK code.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pinPage">
      <layout class="QVBoxLayout" name="pinPageLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="pinBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="pinLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
color: #000000;</string>
         </property>
         <property name="text">
          <string>Enter PIN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>pinInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="pinLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="pinType">
           <property name="styleSheet">
            <string notr="true">font-size: 16px;
color: #000000;
</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="pinInput">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>34</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
padding: 0px 10px;
border: 1px solid #DEE4E9;
border-radius: 2px;
}</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="pinProgress">
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="pinTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pukPage">
      <property name="styleSheet">
       <string notr="true">QLabel { font-size: 16px; color: #000000; }</string>
      </property>
      <layout class="QVBoxLayout" name="pukPageLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="pukTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="pukPageLabel">
         <property name="text">
          <string>PIN1 is locked. Please use PUK to unlock PIN1.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="pukLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="pukLabel">
           <property name="text">
            <string notr="true">PUK</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="pukInput">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>34</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
padding: 0px 10px;
border: 1px solid #DEE4E9;
border-radius: 2px;
}</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pin1Label">
           <property name="text">
            <string notr="true">PIN1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="pin1Input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>34</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
padding: 0px 10px;
border: 1px solid #DEE4E9;
border-radius: 2px;
}</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="pin2Label">
           <property name="text">
            <string>PIN1 again</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="pin2Input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>34</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
padding: 0px 10px;
border: 1px solid #DEE4E9;
border-radius: 2px;
}</string>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="pukBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pinInput</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>log</tabstop>
  <tabstop>messageAgree</tabstop>
  <tabstop>envelopeAgree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
